uint8 id
string name
uint16 palm_IR_sensor
uint16 capacitive_sensor_1
uint16 capacitive_sensor_2
